srcpkgs/sshguard/files/sshguard/run

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Allow the firewall and logger backends to be specified
+[ -f ./conf ] && . ./conf
+
+sv check "${FIREWALL:-iptables}" >/dev/null || exit 1
```

Comment:
This should never have checked for iptables "service", any reasonable setup would use a script like the `runit-iptables` has or `/etc/rc.local` to load firewall rules before network services start.